src/modules/chat/assistant-plan/PlanWithSources.tsx

```diff
@@ -91,7 +101,7 @@ function PlanWithSourcesComponent({ message, inView }: Props) {
   const plan = message.plan ?? planFromSteps;
   const sources = getSourcesWithSteps(plan.steps ?? []);
 
-  const traceData = useBuildTraceData({
+  const { data: traceData, error: traceError } = useBuildTraceData({
```

Comment:
You use data and error here, but traceData and traceError in `TraceDataProvider`. I think we should name it the same way. My personal choice would be just data and error everywhere, but it's your call. :-)

Reply:
I will rewrite the useBuildTraceData to return `traceData` and `traceError`. It's cleaner, then just the error and data properties.